### DISCUSSION TITLE
Welcome to kafka-connect-cosmosdb Discussions!

### DISCUSSION THREAD

**Discussion by ryancrawcour** [microsoft/kafka-connect-cosmosdb#324] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Be respectful and inclusive.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by factor-bytes** [comment]
Hi,

Which connection mode does the connector use to connect with CosmosDB? Direct or Gateway? 

Br
Pardeep

**Reply by ryancrawcour** [reply]
it'll be using the default, which i believe is Gateway
